emr-user-role-mapper-application/src/test/com/amazon/aws/emr/integration/IntegrationTestBase.java

```diff
@@ -0,0 +1,83 @@
+package com.amazon.aws.emr.integration;
+
+import com.google.gson.FieldNamingPolicy;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+/**
+ * Common functionality and constants for integration tests.
+ */
+public class IntegrationTestBase {
+
+  protected static final Gson GSON = new GsonBuilder()
+      .setFieldNamingPolicy(FieldNamingPolicy.UPPER_CAMEL_CASE)
+      .setPrettyPrinting()
+      .create();
+  public static int RELOAD_CFG_TIME_MIN = 1;
+  public static String DEFAULT_MAPPER_IMPL_BUCKET = "urm-integ-test-default-mapper";
+  public static String DEFAULT_MAPPER_IMPL_MAPPING = "default-impl.json";
+  protected static String LOCALHOST_SERVER = "http://localhost";
+  protected static String IMDS_CREDENTIALS_URI = "/latest/meta-data/iam/security-credentials/";
+  protected static String USER_TO_CHANGE = "USER_TO_CHANGE";
+  protected static String USER_ROLE_TO_CHANGE = "USER_ROLE_TO_CHANGE";
+  protected static String AWS_ACCOUNT_TO_CHANGE = "AWS_ACCOUNT_TO_CHANGE";
```

Comment:
Let's make the value as placeholder AWS_ACCOUNT_TO_CHANGE = "{AWS_ACCOUNT_TO_CHANGE}"? Similar to other things that will be substituted.

Reply:
Done.